crates/cargo-util-schemas/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "cargo-util-schemas"
+version = "0.1.0"
+rust-version.workspace = true
+edition.workspace = true
+license.workspace = true
+homepage = "https://github.com/rust-lang/cargo"
```

Comment:
Off-topic question: Why don't we put the homepage and repository in the `workspace.package`?